tests/python/unittest/test_extensions.py

```diff
@@ -148,3 +156,47 @@ def test_subgraph():
     out3 = exe3.forward()
     # check that result matches one executed by MXNet
     assert_almost_equal(out[0].asnumpy(), out3[0].asnumpy(), rtol=1e-3, atol=1e-3)
+
+@unittest.skipIf(check_platform(), "not all machine types supported")
+@unittest.skipIf(is_cd_run(), "continuous delivery run - ignoring test")
```

Comment:
Why skipping this test on CD run?

Reply:
Looks like this was added in this PR #16127 and @rondogency just copied the same from teh other test in teh test_extensions.py file. We're just following the existing standard. If @perdasilva and @szha want to enable it for the CD run they can do that in a future PR

Reply:
ok